Lib/test/test_socket.py

```diff
@@ -1921,6 +1921,7 @@ def testCreateBCMSocket(self):
     def testBindAny(self):
         with socket.socket(socket.PF_CAN, socket.SOCK_RAW, socket.CAN_RAW) as s:
             s.bind(('', ))
+            self.assertIsInstance(s.getsockname(), tuple)
```

Comment:
Is it possible to make the more specific assertion here that `s.getsockname() == ('',)`?

Reply:
I guess so, even though it isn't the greatest of tests to begin with.  I'll make the change anyway.